jwst/ramp_fitting/ramp_fit.py

```diff
@@ -464,7 +465,7 @@ def ols_ramp_fit(model, buffsize, save_opt, readnoise_model, gain_model,
         del var_p4_int
         del var_p4_int2
 
-    del gain_2d
+        del gain_2d   
```

Comment:
This won't work.  By shifting this to the right, you have put it inside the loop over integrations, but it was assigned outside the loop over integrations.

Reply:
I changed it back to the original indentation.